src/Android/Properties/AndroidManifest.xml

```diff
@@ -1,57 +1,48 @@
-<?xml version='1.0' encoding='UTF-8'?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Visual Studio for Mac has a weird issue where simply opening the mobile project completely (and improperly) re-formats the Android manifest.  Unless there are required changes, the entire file should be reverted for the PR.

Reply:
Oh, that's.. interesting :)
I'll fix!